pom.xml

```diff
@@ -5,12 +5,12 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.28</version>
+        <version>3.49</version>
     </parent>
 
     <artifactId>ranorex-integration</artifactId>
     <packaging>hpi</packaging>
-    <version>0.2.1-SNAPSHOT</version>
```

Comment:
The -Snapshot suffix is for local builds and is automatically removed by performing a maven release. So it's actually fine to have the 'Snapshot' in the POM

Reply:
Reverted 'SNAPSHOT' to version.

Reply:
The 0.3.0 was fine, I just meant to add -SNAPSHOT to have 0.3.0-SNAPSHOT.
Sry for beeing unclear